### PR TITLE
Enable ipv6

### PR DESCRIPTION
**Pull request by lwr20**

### CODE REVIEW

File: cookbooks/calico/recipes/general_compute.rb [hunk 1]
```diff
@@ -194,12 +194,13 @@
...
     notifies :restart, "service[bird]", :delayed
 end
+
 template "/etc/bird/bird6.conf" do
     mode "0640"
     source "compute/bird6.conf.erb"
     owner "bird"
     group "bird"
-    notifies :restart, "service[bird]", :delayed
+    notifies :restart, "service[bird6]", :delayed
```
Comment (Lukasa) [inline cookbooks/calico/recipes/general_compute.rb]: This template needs to be extended to pass the 'bgp_peers' variable into it. See the block above.

File: cookbooks/calico/templates/default/compute/bird6.conf.erb [hunk 2]
```diff
@@ -12,42 +12,11 @@
...
-# (as Linux 2.2.x does), you can run multiple instances of the kernel
-# protocol and synchronize different kernel tables with different BIRD tables.
 protocol kernel {
-#       learn;                  # Learn all alien routes from the kernel
+        learn;                  # Learn all alien routes from the kernel
         persist;                # Don't remove routes on bird shutdown
-        scan time 20;           # Scan kernel routing table every 20 seconds
+        scan time 2;           # Scan kernel routing table every 20 seconds
+        device routes;
 #       import none;            # Default is import all
```
Comment (Lukasa) [inline cookbooks/calico/templates/default/compute/bird6.conf.erb]: We need an import filter here, as bird.conf.erb has.

**Comment by lwr20** [comment]
Rats - a bit of playing suggests that Cirros doesn't support DHCPv6.  May need further updates to the chef scripts to install another OS image with does support DHCPv6.  Or else get something on the subnet to support IPv6 RA.

**Comment by neiljerram** [comment]
Yes, definitely.  An Ubuntu snapshot is needed - the IPv6 page at calico-docs/wiki explains how to prepare this.

------------------

**Pull request merged**